app/src/main/java/org/schabi/newpipe/player/helper/PlayerHelper.java

```diff
@@ -267,6 +286,26 @@ public static int getMinimizeOnExitAction(@NonNull final Context context) {
         }
     }
 
+    @BackBehavior
+    public static int getBackBehavior(@NonNull final Context context) {
+        final String action = getPreferences(context)
+                .getString(context.getString(R.string.back_behavior_key),
+                        context.getString(R.string.back_behavior_value));
+        if (action.equals(context.getString(R.string.back_behavior_prev_any_key))) {
+            return BACK_BEHAVIOR_PREV_ANY; // default
+        } else if (action.equals(context.getString(R.string.back_behavior_prev_playlist_key))) {
+            return BACK_BEHAVIOR_PREV_PLAYLIST;
+        } else if (action.equals(context.getString(R.string.back_behavior_hide_key))) {
+            return BACK_BEHAVIOR_HIDE;
+        } else if (action.equals(context.getString(R.string.back_behavior_hide_or_close_key))) {
+            return BACK_BEHAVIOR_HIDE_OR_CLOSE;
+        } else if (action.equals(context.getString(R.string.back_behavior_close_key))) {
+            return BACK_BEHAVIOR_CLOSE;
+        } else {
```

Comment:
This else is useless and can be removed

Reply:
With or without `else` branch I have to write `return BACK_BEHAVIOR_CLOSE_FULLSCREEN` anyway. By using else the code looks better than:
```
...
else if(....)

return BACK_BEHAVIOR_CLOSE_FULLSCREEN
```